kasa/protocols/smartprotocol.py

```diff
@@ -275,9 +277,11 @@ async def _execute_multiple_query(self, requests: dict, retry_count: int) -> dic
                     response, method, raise_on_error=raise_on_error
                 )
                 result = response.get("result", None)
-                await self._handle_response_lists(
-                    result, method, retry_count=retry_count
-                )
+                request_params = rp if (rp := requests.get(method)) else None
```

Comment:
```suggestion
                params = rp if (rp := requests.get(method)) else None
```
to keep it consistent with our naming.

Reply:
There are a few other `params` variables already in the method and I didn't want to risk any future spill over from this variable into other checks for noneness etc.